scalikejdbc-core/src/main/scala/scalikejdbc/TxBoundary.scala

```diff
@@ -41,10 +41,15 @@ object TxBoundary {
     }
   }
 
+  private[scalikejdbc] sealed abstract class TxBoundaryMissingImplicits {
+    implicit def `"To activate TxBoundary.Future, scala.concurrent.ExecutionContext value in implicit scope is required here."`[A]: TxBoundary[A] = sys.error("use internalTxBoundary!")
```

Comment:
Thanks for fixing the message. I'm not sure what the `internalTxBoundary` is. What do you mean?